.eslintrc

```diff
@@ -40,6 +46,73 @@
           }
         ]
       }
-    }
+    },
+    {
+      // Turn off react linting rules for most packages/files 
+      "files": [
+        "spec/**",
+        "app/javascript/packs/**",
+        "app/javascript/packages/address-search/**",
+        "app/javascript/packages/components/**",
+        "app/javascript/packages/compose-components/**",
+        "app/javascript/packages/form-steps/**",
+        "app/javascript/packages/react-hooks/**",
+        "app/javascript/packages/react-i18n/**",
+        "app/javascript/packages/spinner-button/**",
+        "app/javascript/packages/step-indicator/**",
+        "app/javascript/packages/validated-field/**",
+        "app/javascript/packages/verify-flow/**",
+        // In progress: enabling these rules for all files in packages/document-capture
+        "app/javascript/packages/document-capture/context/**", 
+        "app/javascript/packages/document-capture/higher-order/**", 
+        "app/javascript/packages/document-capture/hooks/**",
```

Comment:
Acknowledging that my [earlier comment](https://github.com/18F/identity-idp/pull/9706#discussion_r1419679466) referenced a prior example where we did the same thing of [broad globbing patterns](https://github.com/18F/identity-idp/pull/5083/files#diff-09d428dfddfee8285e0a4c19ae57822ebb38b78c405a75e7f63b721f6fb00f7f) to then return later with [expanded file references](https://github.com/18F/identity-idp/pull/5557/files#diff-09d428dfddfee8285e0a4c19ae57822ebb38b78c405a75e7f63b721f6fb00f7f) once it became more attainable, is that what we're wanting to do here? The downside of these patterns is that it allows someone to add a new file which violates the rules, so there's a risk we trend further away from fixing everything (introduce more issues) vs. resolving everything.

Reply:
I'm really appreciating this conversation. The "Followup Plan" section [here](https://docs.google.com/document/d/1sRvo-qBYKxlg0uo7J2XChI0St3Y3O5GEL0mevJ-9KlM/edit) might help explain my thinking. One thing to note is that after this PR all new files in `packages/document-capture` will be linted with these new rules.

I feel like adding new lint rules can either be a big effort up front or small effort over a long period. I'm trying pretty hard at this moment to avoid the "big effort up front", which means using this kind of overly broad ignore rule.

My hope is that over the next few months a few things will happen:
1. Timnit will manage to enable the lint rules for everything in `packages/document-capture/**` 
2. We'll produce a list of the PRs we did.
3. We'll tell other teams about this and provide a bunch of docs on how to do it without interrupting sprint work.
4. (possibly) I'm hoping to keep chipping away past the `document-capture` package on other packages.

Reply:
Ok, that sounds like a reasonable plan 👍  And I may simply be jaded and cynical in my distrust for follow-through on technical debt clean-up efforts 😅 I think this'll definitely help stem the bleeding so we can make space for ourselves to clean up existing issues.

Reply:
👍 The distrust for follow-through on technical debt clean-up makes perfect sense to me.